### PR TITLE
Mbo/us/aaicddc 35 set up home page

### PR DESCRIPTION
**Pull request by SIbrahimQI**

### CODE REVIEW

File: build/scripts/check-secrets.sh
Comment (ataperekiq) [inline build/scripts/check-secrets.sh:1]: This file needs to keep execute permissions 755. Please revert this change

File: build/scripts/generate-secrets.sh
Comment (ataperekiq) [inline build/scripts/generate-secrets.sh:1]: This file needs to keep execute permissions 755. Please revert this change

File: ReleaseNotes.md
Comment (ataperekiq) [inline ReleaseNotes.md:1]: Did the line endings get changed to Windows (CR-LF)? If so please change back to Unix (LF). In Notepad++, the Edit > EOL Conversion > Unix option

**Review by ataperekiq — CHANGES_REQUESTED** [review]

File: build/scripts/check-secrets.sh
Comment (SIbrahimQI) [reply build/scripts/check-secrets.sh:1]: done

**Review by ataperekiq — APPROVED** [review]
